janusgraph-cql/src/test/java/org/janusgraph/diskstorage/cql/CQLDistributedStoreManagerTest.java

```diff
@@ -54,6 +55,7 @@ public void tearDown() throws BackendException {
     @Override
     @Test
     @FeatureFlag(feature = JanusGraphFeature.OrderedScan)
+    @Disabled(value = "GitHub Actions do run inside of a container")
```

Comment:
if you need to make this change, seems like the new build process (Github Action) is breaking this test. Please add a comment describing why this should be disabled or fix the test so that it does not need to be disabled.

Reply:
GitHub isn't breaking test. Test is just designed to run inside of container. 

For me, it's also not 100% clear. What test tries to test!